pkg/proxy/ipvs/proxier_test.go

```diff
@@ -4295,6 +4295,344 @@ func TestEndpointSliceE2E(t *testing.T) {
 	assert.Nil(t, rsErr2, "Expected no error getting real servers")
 	assert.Len(t, realServers2, 0, "Expected 0 real servers")
 }
+
+// Test_EndpointSliceReadyAndTerminatingLocal tests that when there are local ready and ready + terminating
+// endpoints, only the ready endpoints are used.
+func Test_EndpointSliceReadyAndTerminatingLocal(t *testing.T) {
```

Comment:
I had a similar comment on the iptables tests - it looks like these 2 new test functions share a lot of code/logic/structure. Any way to share some of the overlapping bits could help clarify what is actually changing between them.

Reply:
I had thought about this but the test table would include long lines of output from the iptables buffer which I figured would not be readable enough to be useful.